src/main/java/org/mule/module/apikit/validation/attributes/QueryParameterValidator.java

```diff
@@ -88,7 +88,7 @@ private static void validateQueryParametersStrictly(Map<String, Parameter> query
   }
 
   //only for raml 1.0
-  private static void validateQueryParamArray(String paramKey, Parameter expected, Collection<?> paramValues)
+  static void validateQueryParamArray(String paramKey, Parameter expected, Collection<?> paramValues)
```

Comment:
why not private ?

Reply:
it used to be reused by QueryStringValidator but yes now can be private